nri-config-generator/internal/config/config_test.go

```diff
@@ -30,7 +30,7 @@ transformations:
 func TestGeneratorConfigPortAlreadyInUse(t *testing.T) {
 	dir := t.TempDir()
 	p := path.Join(dir, "config.yml")
-	err := ioutil.WriteFile(p, []byte(text), fs.ModePerm)
+	err := os.WriteFile(p, []byte(text), fs.ModePerm)
```

Comment:
are this changes related to the PR?

Reply:
Semgrep complains of a deprecated API.

Reply:
I addressed them in this separate commit b0ffe2617aa9fe317faa4fbe5c85f03bb6f40f77 as they were small changes.